test/artefacts/marriage-abroad/denmark/ceremony_country/partner_british/opposite_sex.txt

```diff
@@ -40,6 +36,19 @@ If you or your partner have been divorced, widowed or previously in a civil part
 
 ^You’ll also need to provide evidence of nationality or residence if the divorce or dissolution took place outside the UK. You’ll need to get it [legalised](/get-document-legalised) and [translated](/government/collections/list-of-lawyers) if it’s not in English.^
 
+###No trace letters
+
+The Danish authorities will sometimes accept a ‘no trace letter’ instead of a CNI. Check with the Danish authorities to find out if they’ll accept a ‘no trace letter’.
+
+You can get a ‘no trace letter’ from the General Register Office. Ask them to make a search and quote reference ‘SEARCH 123’.
+
+$C
+**General Register Office**\\
+Telephone: +44 (0)300 123 1837 (8am to 8pm Monday to Friday and 9am to 4pm Saturday)
+$C
+
+^Your partner will need to follow the same process and pay the fees to get their own CNI.^
```

Comment:
It's not obvious from the Google Doc that this line should have moved in this outcome. Is it definitely correct that it's moved?